### DISCUSSION TITLE
Working time should not exceed 24hours

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17166] [How to] [answered]
Hello,

I try to create a Working time whose date range is greater than 24 hours and it is not allowing me to create it, I get the message indicated "Working time should not exceed 24 hours"
Is it possible to remove this limitation? how could i do it?
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/975d88a3-b31c-4c0c-81b1-3a098433615f)

I verified it in YetiForce - Latest stable still presents the same message.

**Comment by rskrzypczak** [comment, marked as answer]
it's easy, just uncheck the event
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/f4d81a58-3e77-4193-9f97-5b02a0a6853b)

**Comment by krseguros** [comment]
Thank you!!
